src/components/NavigationBar.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import NavigationBar from "./NavigationBar";
+import { MemoryRouter } from "react-router-dom";
+
+describe("NavigationBar", () => {
+  it("renders a navigation menu", () => {
+    const component = render(
+      <MemoryRouter>

```

Comment:
Please use Enzyme's shallow render instead, and then you don't need to embed the `<NavigationBar />` into a `<MemoryRouter>`.